src/app/notification/notification.service.ts

```diff
@@ -122,6 +122,13 @@ export class NotificationService {
             return;
         }
 
+        // Do not show programming error. They will be logged on the console window
+        if (error instanceof TypeError) {
+            // TODO: create a wiki page and point to the troubleshoot wiki instead
```

Comment:
Did you create a work item of "create a wiki page and point to the troubleshoot wiki"?